core/src/scss/components/card/_card--minimal.scss

```diff
@@ -0,0 +1,22 @@
+.su-card--minimal {
+  @include box-shadow($depth: 'flat', $stroke: 'none');
+  background-color: inherit;
+
+  .su-card__contents {
+    // override modular spacing at all breakpoints
+    @include grid-media-only(all) {
```

Comment:
Could you combine this with the horizontal grid-media-only(all) as well as it does create a lot of CSS and combining them would help optimize the output.

Reply:
The only way I can think to "combine" them is to omit the one for `.su-card-horizontal`. Unfortunately, `.su-card--horizontal`'s padding is more specific, so it wins. So I do in fact need to specify it twice. :(

Reply:
Drat, thought I would ask anyhow.